content/campaigns.md

```diff
@@ -41,12 +41,20 @@ Exclude embedding of page IDs, to reduce payload size. Default: false.
 
     GET https://everydayhero.com/api/v2/campaigns/:id
 
-[View demo in API console](/console/?query=campaigns/au-0.json) 
+or
+
+    GET https://everydayhero.com/api/v2/campaigns/:uuid
+
+[View demo in API console](/console/?query=campaigns/au-0.json)
 
 ### Example
 
     https://everydayhero.com/api/v2/campaigns/au-1.json
 
+or
+
+    https://everydayhero.com/api/v2/campaigns/f6ab24ed-589b-4736-a5bb-aee08d412379.json
```

Comment:
Whadaya think about using the `uuid` from the `au-1` campaign `https://everydayhero.com/api/v2/campaigns/eda1e64c-0001-4000-8000-000000000001.json`?  (only thinking of it as the example link shows a can't be found)